### DISCUSSION TITLE
/posts/vim-essentials-cheatsheet

### DISCUSSION THREAD

**Discussion by giscus** [josean-dev/sveltekit-blog#7] [General]
# /posts/vim-essentials-cheatsheet

This is my Vim cheatsheet with all of the most essential commands! You can check out my youtube video for a complete guide on how almost all of these work and how I go about remembering them.

https://www.josean.com/posts/vim-essentials-cheatsheet

**Comment by Aqa-Ib** [comment]
Thank you so much for this document, it is so useful. 
Bless you!

**Comment by therollingambit** [comment]
This is such a concise yet detailed cheat sheet for commands that we will use 90% of the time! Been following your youtube videos and followed your lazyvim config, just wanna know how do I comment out multiple lines of code?

**Reply by ImtiazKhanDS** [reply]
gc {number of lines you want to comment}

**Comment by sir-george2500** [comment]
Thank you for this

**Comment by pixprocoder** [comment]
I love this. this is such details blog post

**Comment by ghost** [comment]
your vedios  are very good and cleen and informative , and in your vedio the main thing is you don't add any other personal thing here but you try to give more information cleearly , thanks for you're information and you blog , and also don't stop living a blog post for every vedio , which is helpfull

**Comment by EbrahimBonger** [comment]
Great post! Just a small note — there seems to be a typo in the example. The command should be dit (delete inside tag) instead of dit". The trailing quote likely came from the di" command example for quotes.

**Comment by tung-dang** [comment]
Thank you for your great video! Some minors which could improve the video: 

- The pace of video is fast. So it's a little hard to learn all the VIM keys. 
- Some minor mistakes like: going to the top of the file, press `gg` , but the video shows `GG`. The beginner may be confused a little bit.

**Comment by is-ma** [comment]
New follower and fan of your awesome videos. Recently moving from WindowsBloat/VSCode and GUI tools to Debian/i3/Tmux/Nvim and TUI tools.
